index.html

```diff
@@ -1,308 +1,420 @@
 <!DOCTYPE html>
 <html lang="en">
+  <head>
```

Comment:
Dont change the whole formatting of the file. 
1. Keep the formatting same all over the project. Like we don't use any formatters as such. Just the default on in vscode. Better to turn off that as this is making change the whole pattern.
2. It is hard to see the changes you did. I also think, not only the index page, but other pages will also be needed this change.